homage-core/src/main/java/me/oriley/homage/Homage.java

```diff
@@ -192,17 +196,21 @@ public void refreshLibraries() {
             library.setLicense(license);
 
             int iconRes = -1;
-            String icon = library.getLibraryIcon();
-            if (!TextUtils.isEmpty(icon)) {
-                iconRes = getResourceId(mContext, icon, ResourceType.DRAWABLE);
-                if (iconRes <= 0) {
-                    iconRes = getResourceId(mContext, icon, ResourceType.MIPMAP);
-                }
-                if (iconRes <= 0) {
-                    iconRes = android.R.drawable.sym_def_app_icon;
+            String iconUri = library.getLibraryIcon();
```

Comment:
Leave the local field as `icon`, as we don't know if it's a `Uri` or not at this point